client/components/tinymce/plugins/mentions/mentions.jsx

```diff
@@ -0,0 +1,127 @@
+/**
```

Comment:
Thanks! I've got everything configured in Sublime and the linting issues are resolved now.